code/game/objects/structures/fullwindow.dm

```diff
@@ -58,6 +58,39 @@
 		icon_state = "[base_state][junction]"
 		return
 
+/obj/structure/window/full/verb/set_direction() //Full windows get this because it's possible for them to face diagonally
```

Comment:
I went ahead and made a proc that checks whether a mob is living, conscious, and unrestrained, and used that.